app/models/address.rb

```diff
@@ -0,0 +1,6 @@
+class Address < ApplicationRecord
+  has_many :orders
```

Comment:
This would ideally be something more like a `belongs_to :shippable` where `shippable` is a polymorphic relationship representing `Gift`'s and `Order`'s, but I decided to forego that for now to keep the schema changes in the `orders` and `gifts` tables a little smaller - address fields just change to an `address_id`, instead of moving the foreign key over to the `addresses` table